perception/ground_segmentation/src/scan_ground_filter_nodelet.cpp

```diff
@@ -166,8 +166,11 @@ void ScanGroundFilterComponent::classifyPointCloud(
         (points_distance <
          (p->radius * radial_divider_angle_rad_ + split_points_distance_tolerance_));
 
+      float global_slope = std::atan2(p->orig_point->z, p->radius);
       // check points which is far enough from previous point
-      if (
+      if (global_slope > global_slope_max_angle) {
+        p->point_state = PointLabel::NON_GROUND;
```

Comment:
[IMO] Could you add `        calculate_slope = false;` this to the line below? Just for an explicit expression.

Reply:
Thanks for your comment. I have add it.